src/config.rs

```diff
@@ -967,6 +986,19 @@ impl Config {
                 .unwrap_or(false)
     }
 
+    /// The preferred way to run the new tmux session.
+    pub fn tmux_session_attach_mode(&self) -> TmuxSessionAttachMode {
```

Comment:
We should make this function private as well if we want developers to use `.tmux_config()` function

Reply:
done